client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -194,6 +224,39 @@ const PlanOnlyThankYou = ( {
 		},
 	} );
 
+	let upsellProps;
+	if ( promotePlanData ) {
+		upsellProps = {
+			title: translate( 'Advanced tools' ),
+			description: (
+				<>
+					{ translate(
+						'Get the %(planName)s plan for access to WordPress plugins, STFP setup, database credentials, and more, propelling your site to new heights. Upgrade risk-free with our %(refundPeriodDays)s-day money-back guarantee, and pay only the difference ({{s}}%(fullPrice)s{{/s}} %(discountPrice)s).',
+						{
+							comment: 'Upsell for WP Business plan on checkout thank you page',
+							args: promotePlanData,
+							components: { s: <s /> },
+						}
+					) }
+				</>
+			),
+			image: upsellBusinessPlanImage,
+			actions: (
+				<>
+					<Button
+						onClick={ () => {
+							recordTracksEvent( 'calypso_plan_only_thank_you_upgrade_business_click' );
```

Comment:
We should probably specify the name of the page first, so it's consistent with other events triggered by this page:
```suggestion
							recordTracksEvent( 'calypso_thank_you_plan_only_upgrade_to_business_click' );
```